### PR TITLE
📝 Add some precisions to the AI sound marine tutorial

### PR DESCRIPTION
**Pull request by philippart-s**
- add some minors information to help the user to use the tutorial

### CODE REVIEW

File: pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md [hunk 1]
```diff
@@ -19,16 +19,23 @@ The purpose of this tutorial is to show how it is possible to train a model in o
...
 - A user for AI Notebooks
+- Two [containers Object Storage](https://docs.ovh.com/gb/en/storage/pcs/create-container/) to store the data and the model
 - Your own dataset
 
 ## Instructions
 
 ### Uploading your dataset on Public Cloud Storage
 
-If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and create a new object container by clicking `Object Storage`{.action} > `Create an object container`{.action}.
+If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and [create](https://docs.ovh.com/gb/en/storage/pcs/create-container/) a new object container by clicking `Object Storage`{.action} > `Create an object container`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md]: Please include this whole phrase "create a new object container" into the link

File: pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md [hunk 2]
```diff
@@ -19,16 +19,23 @@ The purpose of this tutorial is to show how it is possible to train a model in o
...
 
 ![image](images/new-object-container.png){.thumbnail}
 
+> [!primary]
+>
+> With the OVHcloud Control Panel you can only upload files (not folders). You can, for instance, upload a zip file to optimize the bandwidth and unzip it later when you access it threw your JupyterLab.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md]: Please rephrase:
In the OVHcloud Control Panel, you can upload files but not folders. For instance, you can upload a .zip file to optimize the bandwidth, then unzip it later when accessing it through your JupyterLab.

File: pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md [hunk 3]
```diff
@@ -19,16 +19,23 @@ The purpose of this tutorial is to show how it is possible to train a model in o
...
+> You can also use the OVHcloud AI CLI to upload files and folders (and be more stable than threw your browser).
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md]: Typo: through

File: pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md [hunk 4]
```diff
@@ -49,7 +56,9 @@ ovhai data upload <region> <container> <paths>
 
 You need to attach a volume if your data is in your OVHcloud Object Storage and you want to use it during your experiment. For more information on data, volumes and permissions, see [our guide on data](https://docs.ovh.com/gb/en/publiccloud/ai/cli/access-object-storage-data/).
 
-If you want to launch it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), just follow this [guide](https://docs.ovh.com/gb/en/publiccloud/ai/cli/getting-started-cli/).
+To be able to use the source code below in this article you have to create two containers Object Storage mount as follow:
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md]: 2 Object Storage containers mounted as follows:

**Review by eleapttn — APPROVED** [review]

**Pull request merged**